### DISCUSSION TITLE
CheckMate-withFullRun2

### DISCUSSION THREAD

**Discussion by NishitaDesai** [CheckMATE2/checkmate2#21] [General]
CheckMATE release with new Run2 full data analyses.

<hr /><em>This discussion was created from the release <a href='https://github.com/CheckMATE2/checkmate2/releases/tag/v2.2'>CheckMate-withFullRun2</a>.</em>

**Comment by wk-mike** [comment]
@NishitaDesai   

Hi，thank you very much for the new version.  But when I run the example test code in checkmate2-2.2 : 
> `./CheckMATE -n example  -mgcommand="import model MSSM_SLHA2; generate p p > go go;" -maxev 500 -slha=example_run_cards/auxiliary/testspectrum.slha `

There are some errors: 
> |-> AnalysisHandler 'atlas8tev': Linking to DelphesHandler 'atlas8tev' tree
> ERROR IN FRITZ - MODULE: AnalysisHandler 'atlas8tev'
>                 MESSAGE: could not link all required branches to theDelphesHander 'atlas8tev' equivalents!
> -----> IMMEDIATE ABORT!
> 
> !!! ERROR !!!
> Fritz returned with error. Check logfiles in result folder for more information
> Exiting.

And when I  run this code:
> `./CheckMATE -n example -a 13TeV  -mgcommand="import model MSSM_SLHA2; generate p p > go go;" -maxev 500 -slha=example_run_cards/auxiliary/testspectrum.slha`

There are some errors: 
>  |-> AnalysisHandler 'cms13tev': Analyses updated with sigma = 4407.7 fb an...1
>  *** Break *** segmentation violation
>  Generating stack trace...
> 
> !!! ERROR !!!
> Fritz returned with error. Check logfiles in result folder for more information!
> Exiting.


But when I run this :
> ` ./CheckMATE -n example -a cms13TeV  -mgcommand="import model MSSM_SLHA2; generate p p > go go;" -maxev 500 -slha=example_run_cards/auxiliary/testspectrum.slha`
or
> `./CheckMATE -n example -a atlas13TeV  -mgcommand="import model MSSM_SLHA2; generate p p > go go;" -maxev 500 -slha=example_run_cards/auxiliary/testspectrum.slha`

It can have the right output.

My  CheckMATEv2.2  is compiled with following : 

> ROOT 6.16
> Delphes-3.4.2
> HepMC-2.06.11
> MG5_aMC_v2.7.3
> pythia8245

I want to know what went wrong. 
Thank you very much!

**Reply by AIAS77** [reply]
Hello,
I'm facing the same problem, were you able to fix it?
Thank you!

**Reply by wk-mike** [reply]
No, I can't.  @AIAS77

**Reply by AIAS77** [reply]
Do you have any suggestions?
My professor thinks the issue is from my side and he is losing patience :)
Did you try anything else?
@wk-mike  thank you.

**Reply by wk-mike** [reply]
@AIAS77  You can see this issue #22 
I've been stuck with this problem for a long time. 
I'm really sorry I can't solve this problem either.

**Comment by wk-mike** [comment]
@NishitaDesai 
Hi, 
I want to know if your CheckMATE v2.2  is working properly?
And what's version of these package （ROOT,  Delphes，HepMC，Madgraph，pythia）on your system？

Thank you very much!
